ledger/ledger-api-test-tool-on-canton/unique-contract-keys.conf

```diff
@@ -1 +1,2 @@
 canton.domains.test_domain.domain-parameters.unique-contract-keys = true
+canton.parameters.participant.unique-contract-keys = true
```

Comment:
We now enforce unique contract keys on the participant level. Keeping the above domains config in place for good measure.